### PR TITLE
Dependabot updates

### PR DESCRIPTION
**Pull request by BrianSkibinskiGC**
Prepping for TS 5.2 support in downstream apps

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@gamechanger/eslint-plugin",
-    "version": "0.0.9",
+    "version": "0.0.10",
```
Comment (odielleongc) [inline package.json:3]: should we make it a major version release 1.0.0 since we are upgrading to major release of TS

**Review by eliran — APPROVED** [review]
👍

File: package.json (hunk 1 above)
Comment (eliran) [reply package.json:3]: It depends. I'm not sure what the impact the linter has on the project that uses it. If the project won't have any issues of compatibility, this shouldn't affect it and it doesn't need to be a major version bump. Although, the versioning here doesn't seem to follow semver anyways. (this is a patch update and not an actual minor update)

Comment (BrianSkibinskiGC) [reply package.json:3]: Yeah this repo hasn't really followed semver, but I think this is also a 'supports' and not 'requires' so a patch bump makes sense.

**Pull request merged**